Automattic/newspack-plugin
# Add SCSS linting & formatting

### All Submissions:






### Changes proposed in this Pull Request:



Configures SCSS linting and formatting. Fixes and formats the existing files  ~95% of the changes are generated (`npm run lint:scss -- --fix`), only manual changes were fixing the `scss/selector-no-redundant-nesting-selector` rule (redundant `&` operators)

### How to test the changes in this Pull Request:

1. run `npm run lint:scss` - should exit without errors
1. run `npm run format:scss` - should not change any files

### Other information:
